### PR TITLE
Feature/acl

### PR DESCRIPTION
**Pull request by sureshvarman-cronj**

### CODE REVIEW

File: src/server/utils/acl.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
+  "onboarding/onboarding-mgmt": {
```
Comment (gr4per) [inline src/server/utils/acl.json]: acl.json format differs from spec

File: src/server/utils/acl.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    },
+    "resources": [
+      {
+        "type": "rest",
+        "resourceId": "/api/getWorkflowTypes",
+        "actions": ["view"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "/api/campaigns/start/[%w | %p]*",
```
Comment (gr4per) [inline src/server/utils/acl.json]: syntax not defined in spec

File: src/server/utils/acl.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        "actions": ["view"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "/edit/[%w | %p]*",
+        "actions": ["view"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "/edit/[^/]+/contacts",
```
Comment (gr4per) [inline src/server/utils/acl.json]: syntax not defined in spec

File: src/server/utils/acl.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
+{
+  "onboarding/onboarding-mgmt": {
+    "translations": {
+      "name" : {
+        "en": "On-boarding portal management",
```
Comment (gr4per) [inline src/server/utils/acl.json]: This should be "Define and execute onboarding campaigns"
de: "Erstellen und Ausführen von Kampagnen"

File: src/server/utils/acl.json [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+        "de": ""
+      },
+      "description": {
+        "en": "Management box to manage the onboarding stuffs",
```
Comment (gr4per) [inline src/server/utils/acl.json]: This is far too lax. "Onboarding management includes all administrative functions for campaign management, such as Campaign creation, definition, content templating and execution"
de "Enthält die administrativen Funktionen zur Kampagnenverwaltung, wie z.B. Erstellen, Inhaltsvorlagen, Definition von Empfängern etc."

File: src/server/utils/getServerConfig.js [hunk 6]
```diff
@@ -1,4 +1,6 @@
 const _ = require('lodash');
+const bouncer = require('ocbesbn-bouncer');
```
Comment (gr4per) [inline src/server/utils/getServerConfig.js]: why are we still using ocbesbn?
all our modules have to be opuscapita scope.
Please dont create any new one with ocbesbn

Comment (sureshvarman-cronj) [inline src/server/utils/getServerConfig.js]: Okay, no future modules will be of this name, but let this be on the same name. Thought of name consistent so did as ocbesbn-bouncer

**Pull request merged**